jobboard/tests.py

```diff
@@ -112,3 +134,51 @@ def test_get_jobs_by_major():
     # tests function by two majors "Major.COMPUTER_SCIENCE" and "Major.LAW"
     assert jobs_by_computer_science_and_law_major_from_test_function == \
            jobs_by_computer_science_and_law_major_added_manually
+
+
+@pytest.mark.django_db
+def test_page_display_to_user_that_is_not_student_and_not_recruiter(client):
+    example_user = User.objects.create_user("username", "password")
+    client.force_login(example_user)
+    response = client.get("/")
+    assert response.status_code == 200
+    assertTemplateUsed(response, 'jobboard/board.html')
+    assert response.context['user'] == example_user
+    assert response.context['user_is_student'] is False
+    assert response.context['user_is_recruiter'] is False
+    date_six_months_ago = date.today() - timedelta(days=180)
+    jobs_to_display = Job.get_jobs_posted_on_or_after_specific_date(date_six_months_ago)
+    # test that jobs displayed are only jobs created 6 months ago
+    assert response.context['jobs_to_display'] == jobs_to_display
+
+
+@pytest.mark.django_db
+def test_page_display_to_user_that_is_student(client, example_student_a):
+    client.force_login(example_student_a.user)
+    response = client.get("/")
+    assert response.status_code == 200
+    assertTemplateUsed(response, 'jobboard/board.html')
```

Comment:
This is not the pytest-style of doing this, you could just assert on `response.template_name` instead.